tests/common/plugins/sanity_check/recover.py

```diff
@@ -26,13 +27,54 @@ def reboot_dut(dut, localhost, cmd):
         assert False, "Failed to reboot the DUT"
 
     localhost.wait_for(host=dut.hostname, port=22, state="started", delay=10, timeout=300)
-    wait(30, msg="Wait 30 seconds for system to be stable.")
+    wait(120, msg="Wait 120 seconds for system to be stable.")
 
 
-def recover(dut, localhost, recover_method):
+def __recover_interfaces(dut, fanouthosts, result):
+    for port in result['down_ports']:
+        logging.info("Restoring port {}".format(port))
+        fanout, fanout_port = fanout_switch_port_lookup(fanouthosts, port)
+        if fanout and fanout_port:
+            fanout.no_shutdown(fanout_port)
+        dut.no_shutdown(port)
+    wait(30, msg="Wait 30 seconds for interface(s) to restore.")
```

Comment:
How do we ensure that this fixed the issue and we don't have to go in for a config reload or reboot?

Reply:
Great question!

I believe that if we cannot bring it back up with enabling these 2 ports. Then we won't be able to recover it with config reload either unless something else is wrong. But if something else is wrong, then we should have other sanity check failures that would trigger either config reload or reboot in this adaptive sequence.